library/portable-simd/crates/core_simd/src/masks.rs

```diff
@@ -389,6 +390,10 @@ where
     T: MaskElement,
     LaneCount<N>: SupportedLaneCount,
 {
+    /// Converts a SIMD mask to an array of bools.
+    ///
+    /// ## Cost
+    /// Copies the bytes of the array
```

Comment:
incorrect, misleading, and not useful, as the real cost is the usage of instructions for converting the mask types to the boolean values, which can be extremely expensive relative to merely moving the bytes around.

Reply:
it's still _relatively_ cheap, but merely _moving bytes_ can easily be, well, the easy part, and for people who care about the cost of something done in SIMD, it can be one of the highest costs in their program!